mistral/bitstream.cc

```diff
@@ -387,6 +389,13 @@ void Arch::build_bitstream()
 {
     MistralBitgen gen(getCtx());
     gen.run();
+    // This is a hack to run timing analysis yet again after the bitstream is
+    // configured in Mistral, because the analogue simulator won't work until
+    // it has a bitstream in the library.
+    //
+    // A better solution would be to move a lot of this bitstream code to
+    // {un,}bind{Bel, Pip} and friends, but we're not there yet.
+    timing_analysis(getCtx(), true, true, true, true, true);
```

Comment:
Can you add a `log_info` statement here along the lines of "Running signoff timing analysis...", to explain what it's doing and the reason for getting a second Fmax at the end?